pslab/external/MLX90614.py

```diff
@@ -54,15 +48,15 @@ def getRaw(self):
             return False
 
     def getObjectTemperature(self):
-        self.source = self.OBJADDR
```

Comment:
Here we could use the `select_source` method for improved readability.

Reply:
Good point, thank you!